docs/users.rst

```diff
@@ -307,7 +307,8 @@ automatically:
 - ``architecture``: Architecture for the image.
 - ``com.redhat.build-host``: OpenShift node where image was built.
 - ``vcs-ref``: A reference within the version control repository; e.g. a git commit.
-- ``vcs-type``: The type of version control used by the container source. Currently, only git is supported.
+- ``vcs-type``: The type of version control used by the container source. Currently, only git is
+    supported.
```

Comment:
For some reason the line is now formatted in **bold** https://osbs--204.org.readthedocs.build/en/204/users.html?highlight=vcs-type#writing-a-dockerfile